pkg/ddevapp/ddevapp_test.go

```diff
@@ -222,6 +222,19 @@ var (
 			Safe200URIWithExpectation:     testcommon.URIWithExpect{URI: "/README.txt", Expect: "This is a simple readme."},
 			DynamicURI:                    testcommon.URIWithExpect{URI: "/index.php", Expect: "This program makes use of the Zend Scripting Language Engine"},
 		},
+		// 12: Silverstripe
```

Comment:
Please don't inject this here and break the knowledge of all things that come after it, please put it as a new entry at the end of the list, thanks. This will be 17.

Reply:
:+1:
I went alphabetically :)

Reply:
And I see I even failed at basic alphabet! 😆